.github/workflows/ci.yml

```diff
@@ -0,0 +1,166 @@
+name: Continuous Integration
+on:
+  pull_request:
+  merge_group:
+    types: [checks_requested]
+  workflow_dispatch:
+
+jobs:
+  elixir:
+    uses: ./.github/workflows/elixir.yml
```

Comment:
I hate the fact that you can't move those to a separate directory, they are never started directly anyways.

Reply:
It looks like they're also not being depended on properly in the `depends_on` specifier -- build-images start running immediately :-/

Reply:
Which I think might be OK for now. PR merges and CD pipeline will still wait for everything to turn green. It also may be good to see CI runs where unit tests have failed _but also_ the image failed to build.